bot-components/Bot_info.mli

```diff
@@ -0,0 +1,6 @@
+type bot_info =
```

Comment:
If you create a module dedicated to a type, the OCaml convention is then to call this type `t`.